documentation/asciidoc/accessories/build-hat/net-brick.adoc

```diff
@@ -104,7 +104,7 @@ void MotorPropertyEvent(object? sender, PropertyChangedEventArgs e)
 
 ==== Waiting for initialization
 
-The brick can take long before it initializes. A wait for sensor to be connected has been implemented,
+The brick can take long before it initializes. A wait for a sensor to be connected has been implemented,
```

Comment:
The comma at the end of the line should probably be a colon for a full-stop?

Reply:
fixed.